integration/tools/validation/src/main/java/alluxio/cli/fuse/CorrectnessValidationOptions.java

```diff
@@ -0,0 +1,163 @@
+package alluxio.cli.fuse;
+
+import com.google.common.base.Preconditions;
+import org.apache.commons.cli.CommandLine;
+import org.apache.commons.cli.CommandLineParser;
+import org.apache.commons.cli.DefaultParser;
+import org.apache.commons.cli.Option;
+import org.apache.commons.cli.Options;
+import org.apache.commons.cli.ParseException;
+
+/**
+ * This class stores the options passed in from CLI.
+ */
+public final class CorrectnessValidationOptions {
+
+  public static final CommandLineParser PARSER = new DefaultParser();
+  private final String mLocalDir;
+  private final String mFuseDir;
+  private final CorrectnessValidationOperation mOperation;
+  private final int mNumThreads;
+  private final int mNumFiles;
+
+  /**
+   * Constructs a {@CorrectnessValidationOptions} object through CLI.
+   * @param args arguments from CLI
+   * @return a {@CorrectnessValidationOptions} holding all the test options
+   * @throws ParseException
+   * @throws IllegalArgumentException
+   */
+  public static CorrectnessValidationOptions createOptions(String[] args)
+      throws ParseException, IllegalArgumentException {
```

Comment:
Runtime exceptions
- easy to understand error message
- should be caught in the last step or thrown to user directly, no need to add to the method signature